x/gov/handler_test.go

```diff
@@ -1005,9 +968,24 @@ func TestTally(t *testing.T) {
 			},
 			ExpResult: Proposal_Rejected,
 		},
-		"Updates an electorate on success": {
+		"Updates latest electorate on success": {
```

Comment:
It would be good for a similar test where the proposal started with version 1 (add A)
It was updated to version 2 in the meantime (add X+Y).
And then proposal passes with a threshold that passes with version 1, but fails in version 2
And then the final result includes A, X and Y. All three.

If I understand, this is a high-level integration test to over the main issue that was to be addressed with versioning

Reply:
yes, this would be a good test case covering the whole flow.